src/entities/entities.test.ts

```diff
@@ -1,16 +1,15 @@
 import JSBI from 'jsbi'
 import invariant from 'tiny-invariant'
-import { TradeType, Token, CurrencyAmount } from '@uniswap/sdk-core'
-import { WONE as _WONE5 } from '@zuzu-cat/defira-sdk-core'
+import { WETH9 as _WETH9, TradeType, Token, CurrencyAmount } from '@uniswap/sdk-core'
```

Comment:
Why was all this reverted?

Reply:
Use @uniswap/sdk-core data types instead of defira-sdk-core.  We might end up deleting defira-sdk-core. AFAIK nothing in defira-sdk should be using defira-sdk-core data types

Reply:
Let's keep those changes for now. If anything we might move the ONE data type here from that package. It doesn't make sense to use ETH for a Harmony SDK